etc/mirror-box.ts

```diff
@@ -16,7 +17,25 @@ export async function mirror(name: string) {
   });
 }
 
+export async function mirrorDatabase() {
+  const start = process.hrtime();
+  const spinner = ora(`${chalk.yellowBright("Mirroring Database")}`).start();
+  spinner.start(`${chalk.yellowBright("Fetching next data...")}`);
+
+  fetchDatabase()
```

Comment:
We shouldn't perform the database fetching here, because this script is meant to download the prefetched JSON file.

Please reuse the `mirror` function instead, just replicate what we have been doing for the other JSON files.

Reply:
hai mas @zainfathoni already fix this.
sorry for miss understanding about this. 

kindly check it again mas.
Thanks